go.mod

```diff
@@ -291,7 +290,7 @@ require (
 	deps.dev/api/v3 v3.0.0-20240923041156-0312db85d6d6
 	entgo.io/contrib v0.6.0
 	entgo.io/ent v0.14.0
-	github.com/99designs/gqlgen v0.17.54
+	github.com/99designs/gqlgen v0.17.55
```

Comment:
This should be up-to date on the main branch so we can remove this and this will remove all the generated code in this PR.